fcrepo-webapp/src/main/resources/spring/master.xml

```diff
@@ -3,14 +3,13 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="
     http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
-    
+
     <!-- Master context for fcrepo4. -->
 
     <import resource="${fcrepo.spring.repo.configuration:classpath:/spring/repo.xml}"/>
     <import resource="${fcrepo.spring.rest.configuration:classpath:/spring/rest.xml}"/>
-    <import resource="${fcrepo.spring.minter.configuration:classpath:/spring/minter.xml}"/>
```

Comment:
This is an important point: we should certainly make it easy to customize minting behavior. What would you think about adding the minter module and related configuration to `fcrepo-webapp-plus`?